Confus/ConfusServer/Networking/Connection.cpp

```diff
@@ -81,5 +87,24 @@ namespace ConfusServer
 			a_InputStream.Read(contents);
 			std::cout << "Message received: " << contents << std::endl;
 		}
+
+        void Connection::processPlayerPacket(RakNet::BitStream& a_InputStream)
+        {
+            bool isHeavyAttack, isMainPlayer;
```

Comment:
I generally wouldn't recommend combining two or more declarations. The syntax is quite confusing once you do such declarations for pointers and references, so you should be wary of making a habbit out of it